client/src/pages/ListPage.js

```diff
@@ -3,63 +3,65 @@ import "./ListPage.css";
 import { ReactComponent as ForwardButtonIcon } from "../svg/icon-chevron-right.svg";
 import { ReactComponent as DeleteIcon } from "../svg/icon-delete.svg";
 // import requirements
-import { Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
 
 export default function ListPage({
   inputValue,
   globalReminders,
   deleteReminder,
 }) {
-  function renderList() {
-    return globalReminders
-      .filter((reminder) => {
-        return (
-          reminder.trigger.toLowerCase().includes(inputValue.toLowerCase()) ||
-          reminder.tasks.some((task) =>
-            task.with.toLowerCase().includes(inputValue.toLowerCase())
-          )
-        );
-      })
-      .map((reminder, index) => {
-        return (
-          <section
-            className={`Overview__card dispFlex ${
-              !reminder.trigger.toLowerCase().includes(inputValue.toLowerCase())
-                ? "withHighlight"
-                : ""
-            } `}
-            key={index}
-          >
-            <div className="Overview__card__number">
-              {reminder.tasks.length}
-            </div>
-            <div className="Overview__card__txt">
-              <p> In case of:</p>
-              <h4>{reminder.trigger}</h4>
-              <h5>{reminder.triggerEvent}</h5>
-              {!reminder.trigger
-                .toLowerCase()
-                .includes(inputValue.toLowerCase()) && (
-                <p>Found as optional party</p>
-              )}
-            </div>
-            {reminder.tasks.length > 0 ? (
-              <Link
-                to={`/overview/${reminder.reminderId}/0`}
-                className="Overview__card__link"
-              >
-                <ForwardButtonIcon className="icon--dark opaque" />
-              </Link>
-            ) : (
-              <DeleteIcon
-                onClick={() => deleteReminder(reminder.reminderId)}
-                className="Overview__card__link icon--dark opaque"
-              />
+  const history = useHistory();
+  const renderList = globalReminders
+    .filter((reminder) => {
+      return (
+        reminder.trigger.toLowerCase().includes(inputValue.toLowerCase()) ||
+        reminder.tasks.some((task) =>
+          task.with.toLowerCase().includes(inputValue.toLowerCase())
+        )
+      );
+    })
+    .map((reminder, index) => {
+      return (
+        <section
+          className={`Overview__card dispFlex ${
+            !reminder.trigger.toLowerCase().includes(inputValue.toLowerCase())
+              ? "withHighlight"
+              : ""
+          } `}
+          key={index}
```

Comment:
Using the `index` as a key is an antipattern. You can use it, but why not use something more unique. Maybe you finde something in the `reminder` to use that as a key.

Reply:
* Wink Wink*